jenkins-builds/7.0.UP00-19.03-gpu

```diff
@@ -15,6 +15,7 @@
  GSL-2.5-CrayCCE-19.03.eb
  GSL-2.5-CrayIntel-19.03.eb
  HDFView-2.14.eb                                    --set-default-module
+ Horovod-0.16.0-CrayGNU-19.03-tf-1.12.0.eb          --set-default-module
```

Comment:
@rsarm Please remove the file `7.0.UP00-19.03-gpu` from your pull request: we will update the production list separately once the recipe will be merged in the repository (see #1081)